docs/imviz/plugins.rst

```diff
@@ -232,10 +236,9 @@ All images are automatically linked by pixels on load but you can use
 it to re-link by pixels or WCS as needed.
 
 For WCS linking, the "fast approximation" option uses an affine transform
-to represent the offset between images, if possible. It is much more
-performant at the cost of accuracy but should be accurate to within a pixel
-for most cases. If approximation fails, WCS linking still automatically
-falls back to full transformation.
+to represent the offset between images, if possible. This method, although less accurate,
+is much more performant but should be accurate to within a pixel for most cases.
+If approximation fails, WCS linking will fall back to the full transformation.
```

Comment:
```suggestion
to represent the offset between images, if possible. This method, although less accurate,
is much more performant and should still be accurate to within a pixel for most cases.
If approximation fails, WCS linking will fall back to the full transformation.
```
I didn't like that the "but" read like it was contrasting "performant" rather than "less accurate."